pipelines/controller/src/test/java/org/openmrs/analytics/DataPropertiesTest.java

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright 2020-2023 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.openmrs.analytics;
+
+import org.junit.Assert;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+
+@SpringBootTest
```

Comment:
Questions: Do we need to use `@SpringBootTest` only because of "Autowiring" of `dataProperties`? Also should we have a separate config file for test-purposes only (i.e., under `src/test`)?

Reply:
Is there any challenge with having @SpringBootTest? Right now, the tests written are not validating on the configs but on certain formats. Sure, we can add a config file under `src/test` when required.